### PR TITLE
[Bug] [seatunnel-connectors] Age is too far from real when using fake.

### PR DESCRIPTION
**Pull request by zzzzzzzs**
https://github.com/apache/incubator-seatunnel/issues/1725

## Purpose of this pull request
Fixed the problem that the age is too different from the real one when using fake, and changed the system time in the previous code to a random number within 100.

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/fake/source/FakeSourceStream.java [hunk 1]
```diff
@@ -67,12 +68,14 @@ public String getPluginName() {
...
     private static final String[] NAME_ARRAY = new String[]{"Gary", "Ricky Huo", "Kid Xiong"};
+    private static final int AGE_LIMIT = 100;
 
     @Override
     public void run(SourceContext<Row> ctx) throws Exception {
+        Random random = new Random();
         while (running) {
             int randomNum = (int) (1 + Math.random() * NAME_ARRAY.length);
-            Row row = Row.of(NAME_ARRAY[randomNum - 1], System.currentTimeMillis());
+            Row row = Row.of(NAME_ARRAY[randomNum - 1], (long) random.nextInt(AGE_LIMIT));
```
Comment (ruanwenjun) [inline seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/fake/source/FakeSourceStream.java:78]: Use int here is ok, you don't need to convert to long. You can change the type in line 51

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

**Comment by xleoken** [comment]
thanks @zzzzzzzs, please modify `LONG_TYPE_INFO` to `INT_TYPE_INFO`, L51

**Comment by CalvinKirs** [comment]
no updated, closed. if you have any updates, you can leave msg, I will reopen it.

**Comment by zzzzzzzs** [comment]
> no updated, closed. if you have any updates, you can leave msg, I will reopen it.

Hello, it has been changed from long to int, please check it out.

**Pull request closed without merging**